<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCMNormalEstimationDialog</class>
 <widget class="QDialog" name="VCMNormalEstimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Normal estimation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Offset Radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_inputOffsetRadius">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Convolution radius:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="m_inputConvolveRadius">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Convolution Neighbors:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_inputConvolveNeighbors">
       <property name="suffix">
        <string> neighbors</string>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_inputUseConvRad">
     <property name="text">
      <string>Use Convolution Radius</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>buttonBox</zorder>
  <zorder>m_inputOffsetRadius</zorder>
  <zorder>m_inputConvolveRadius</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>m_inputConvolveNeighbors</zorder>
  <zorder>label_2</zorder>
  <zorder>m_inputUseConvRad</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VCMNormalEstimationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VCMNormalEstimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
